objcxx_eh.cc

```diff
@@ -515,14 +519,71 @@ static void eh_cleanup(void *exception)
 	objc_release(*(id*)exception);
 }
 
+namespace __cxxabiv1
+{
+    struct __cxa_exception
```

Comment:
We used to have a definition like this, but we removed it because different C++ runtimes had subtly different variations on it.  This is why we have the thing that dynamically probes it for the fields that we need.  I'm not sure that reintroducing it is a good idea.